src/PriceOracle.sol

```diff
@@ -25,16 +25,15 @@ contract PriceOracle is AccessControl {
     uint64 public lastAdjustedRound;
 
     // The minimum price allowed
-    uint32 public minimumPrice = 1024;
+    uint32 public minimumPrice = 24000;
```

Comment:
I would do the same as for `currentPrice`

Reply:
will do that, makes sense, and looks better. thnx